synapse/config/experimental.py

```diff
@@ -173,3 +173,8 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
         self.msc3952_intentional_mentions = experimental.get(
             "msc3952_intentional_mentions", False
         )
+
+        # MSC3959: Do not generate notifications for edits.
+        self.msc3958_supress_edit_notifs = experimental.get(
+            "msc3958_supress_edit_notifs", False
+        )
```

Comment:
This has a typo: supress -> sup**p**ress

Reply:
That's annoying, but I think I'm going to just leave it since it is a flag that will hopefully disappear soon.